src/network-interface.js

```diff
@@ -4,7 +4,10 @@ import { extractRequestFiles } from './helpers'
 export class HTTPUploadNetworkInterface extends HTTPFetchNetworkInterface {
   fetchFromRemoteEndpoint({ request, options }) {
     // Skip upload proccess if SSR
-    if (typeof window !== 'undefined') {
+    if (
+      typeof window !== 'undefined' ||
+      (typeof global !== 'undefined' && typeof global.FormData !== 'undefined')
+    ) {
```

Comment:
No if just `if (FormData) {` wil cause a `ReferenceError: FormData is not defined`